### DISCUSSION TITLE
Problem on Using Wavefront

### DISCUSSION THREAD

**Discussion by v-nafiseh** [spring-petclinic/spring-petclinic-cloud#60] [Q&A] [answered]
I want to deploy the project on K8s based on [README](https://github.com/spring-petclinic/spring-petclinic-cloud/tree/master#setting-things-up-in-kubernetes)
I applied `kubectl apply -f k8s/init-namespace/`, and want to create the Wavefront secret with the next command: 
`kubectl create secret generic wavefront -n spring-petclinic --from-literal=wavefront-url=https://wavefront.surf --from-literal=wavefront-api-token=2e41f7cf-1111-2222-3333-7397a56113ca`
But, I don't have my own api token for Wavefront. I am wondering if there's a free plan of Wavefront and I could use it for running this project.

**Comment by arey** [comment]
Hi @odedia did you have any idea if a free plan for Wavefront exists? And if not, how can you bypass Wavefront?

**Comment by odedia** [comment, marked as answer]
Free plan should still exist from what I recall. 
you can disable wavefront integration with 

management.metrics.export.wavefront.enabled=false
